batch/data-plane/target/test-classes/session-records/canCRUDJobSchedule.json

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
Files in /target/ shouldn't be committed to the repo, should they?

Reply:
This version might be outdated.
I changed it to protected last week.

For target files, they need to be there. 
Those are the recorded test files. Used during Playback / CI testing.

Reply:
I don't believe this is correct. These resources should be elsewhere inside src and then copied into target - because `mvn clean` should erase the target directory.

Reply:
I believe Vinay was following keyvault pattern which itself is wrong. They can be under src/resources, as it is done in azure-libraries-for-java repo.

Reply:
These files won't get deleted.
Currently, maven clean plugin is configured to 
  <exclude>test-classes/session-records/</exclude>
In our pom.client.xml.
This applies to Keyvault as well.

Reply:
So, we should standardize this and keep the files in resources, and copy them over to target/session-records/  during build.
Apply to keyvault too.

Reply:
Yes